BookingsAPI/Bookings.DAL/Commands/UpdateSuitabilityAnswersCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Bookings.DAL.Commands.Core;
+using Bookings.DAL.Exceptions;
+using Bookings.Domain;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bookings.DAL.Commands
+{
+    public class UpdateSuitabilityAnswersCommand : ICommand
+    {
+        public Guid ParticipantId { get; set; }
+        public Guid HearingId { get; set; }
+        public IList<SuitabilityAnswer> SuitabilityAnswers { get; set; }
+        public UpdateSuitabilityAnswersCommand(Guid hearingId, Guid participantId, IList<SuitabilityAnswer> suitabilityAnswers)
+        {
+            ParticipantId = participantId;
+            HearingId = hearingId;
+            SuitabilityAnswers = suitabilityAnswers;
+        }
```

Comment:
We need a created date for suitability answer.

Reply:
Fixed.